src/client/styles/dashboard-table.scss

```diff
@@ -0,0 +1,85 @@
+$icon-size: 18px;
+$icon-up: "\25b2";
+$icon-down: "\25bc";
+
+.dashboard-table {
```

Comment:
for BEM - if the `Block` does not have it's own file, I will put all the declarations within one .dashboard-table {..}. If the Block has its own file (like this one), I will split the declarations by element.

Reply:
Out of curiosity, why didn't you nest everything inside `.dashboard-table {`?

Reply:
I would if I put this in styles.scss. It would be a way of "insulating" or separating .dashboard-table `Block` away from the rest of the styles. "These are all of the things to do with .dashboard-table". 

Seeing as I'm already in dashboard-table, I can assume everything in here is to do with dashboard-table and doesn't need to be insulated in the same way. So I've split everything by element. In this case I'm insulating the elements from the other ones. If you were writing BEM in css, you would write it per block and element (and modifier) too. 


Although granted, it doesn't actually make that much of a difference. Just a mental thing to keep all the elements separated. For large frontend projects, typically the elements would get their own file as well (especially if there are a lot of modifiers), not that I've worked on one on that scale for a long time 🙂 

I guess partly it's also because I like the approach of CSS modules. 1 file for the block/parent component, then each element/thing has its own declaration. 

I'm not too fussed about `DRY` in terms of repeating .dashboard-table { .. } when it comes to (s)css. what's more important is that .dashboard-table* does not interfere with other styles, and that it can be reused and/or modified. (and to be honest, these should probably just all be modifiers to govuk-table). 

tl;dr  - preference 🙂